--- archive.php	2005-01-04 06:20:06.000000000 +0100
+++ ../../../../a6/wp-content/themes/default/archive.php	2005-03-23 13:19:00.000000000 +0100
@@ -35,6 +35,20 @@
 		</div>
 
 		<?php while (have_posts()) : the_post(); ?>
+
+        <?php if (is_mini_post()) : ?>
+	
+        <div class="minipost">
+            <p>
+            <small><?php the_time('F jS, Y') ?></small><br />
+            <a href="<?php the_permalink() ?>" rel="bookmark" title="Permanent Link to <?php the_title(); ?>">#</a>
+            <?php the_content() ?>
+            <?php comments_popup_link("(0)", "(1)", "(%)"); ?>
+            </p>
+		</div>
+	
+		<?php else : ?>
+
 		<div class="post">
 				<h3 id="post-<?php the_ID(); ?>"><a href="<?php the_permalink() ?>" rel="bookmark" title="Permanent Link to <?php the_title(); ?>"><?php the_title(); ?></a></h3>
 				<small><?php the_time('l, F jS, Y') ?></small>
@@ -43,13 +57,15 @@
 					<?php the_excerpt() ?>
 				</div>
 		
-				<p class="postmetadata">Posted in <?php the_category(', ') ?> <strong>|</strong> <?php edit_post_link('Edit','','<strong>|</strong>'); ?>  <?php comments_popup_link('No Comments &#187;', '1 Comment &#187;', '% Comments &#187;'); ?></p> 
+				<p class="postmetadata">Posted by <?php the_author() ?> <strong>|</strong> <?php edit_post_link('Edit','','<strong>|</strong>'); ?>  <?php comments_popup_link('No Comments &#187;', '1 Comment &#187;', '% Comments &#187;'); ?></p> 
 				
 				<!--
 				<?php trackback_rdf(); ?>
 				-->
 			</div>
 	
+        <?php endif; ?>
+
 		<?php endwhile; ?>
 
 		<div class="navigation">
